app/assets/javascripts/application/evaluationtable.js

```diff
@@ -4,8 +4,8 @@ var evaluationTable = (function($) {
       hiddenUsersList = null;
 
   module.init = function() {
+    if (typeof initEvaluationTable === 'undefined') { return; }
```

Comment:
We used same approach for passing js variables from [templates](https://github.com/kir-dev/pek-next/blob/c1eba4257e4353609be0e49cc0ec6740c61bd93b/app/views/evaluations/table.html.erb#L120), but it doesn't seem OK to me. I welcome any better solution.

Reply:
I've found a [better solution](
https://guides.rubyonrails.org/asset_pipeline.html#controller-specific-assets), but it would be nicer to refactor everywhere. I've mentioned I am working on webpack integration too, so I would do it there. 👍 if it is acceptable.